src/main/org/firebirdsql/jdbc/FBBlob.java

```diff
@@ -448,7 +448,7 @@ boolean isNew() {
         return isNew;
     }
 
-    GDSHelper getGdsHelper() {
+    public GDSHelper getGdsHelper() {
```

Comment:
Ok thanks. I'll look at that to see what I should take into account for future changes. I don't think passing the `FBConnection` around would be the right thing, and maybe keeping `GDSHelper` is necessary.